plugins/org.eclipse.emf.mwe2.lib/src/org/eclipse/emf/mwe2/ecore/EcoreGenerator.java

```diff
@@ -64,7 +65,14 @@ public void setLineDelimiter(String lineDelimiter) {
 	private String genModel;
 	
 	private ResourceSet resourceSet;
-	
+
+	/**
+	 * @since 2.11.4
```

Comment:
In fact we'd need to update to 2.12 if we introduce new API. What do you think @cdietrich ?

Reply:
as this is optional and compatible in would keep 2.11.4